gnovm/pkg/gnolang/helpers.go

```diff
@@ -35,6 +36,11 @@ func IsStdlib(s string) bool {
 	return !strings.HasPrefix(s, "gno.land/")
 }
 
+// IsRemotePkgPath determines whether s is a remote pkg path, i.e.: not a filepath nor a standard library
+func IsRemotePkgPath(s string) bool {
```

Comment:
I'd say we can keep this unexported at the caller level, and not introduce it as a helper

Reply:
52d8d965f0479a0f050db716b697b44fbfd3e919